### PR TITLE
More prep for 7.7.0

### PR DESCRIPTION
**Pull request by hjoliver**
* moved the last file out of `conf/` and into `etc/` - `job-init-env.sh`

### CODE REVIEW

File: bin/cylc-view [hunk 1]
```diff
@@ -23,8 +23,7 @@ editor, after optional include-file inlining and Jinja2 preprocessing.
 
 The edit process is spawned in the foreground as follows:
   % <editor> suite.rc
-Where <editor> is defined in the cylc site and user config files
-($CYLC_DIR/conf/global.rc and $HOME/.cylc/global.rc).
+Where <editor> can be specific in cylc global config.
```
Comment (oliver-sanders) [inline bin/cylc-view]: Where <editor> can be ~specific~ *set* in cylc global config.

Comment (hjoliver) [reply bin/cylc-view]: oops, was prob meant to be 'specified'; but set is better...

File: bin/cylc-scan [hunk 2]
```diff
@@ -221,13 +221,17 @@ def main():
     if options.all_ports:
         args.extend(glbl_cfg().get(["suite host scanning", "hosts"]))
     if not args:
+        print 1, args
```
Comment (oliver-sanders) [inline bin/cylc-scan]: Did you mean to add this?

Comment (hjoliver) [reply bin/cylc-scan]: ah, bollocks, no.  Some debug left-over from the cylc scan issues today.  The dangers of rushing...

**Pull request merged**